src/screens/application/container/applicationContainer.js

```diff
@@ -854,7 +882,6 @@ export default connect(
     // Application
     isDarkTheme: state.application.isDarkTheme,
     selectedLanguage: state.application.language,
-    notificationSettings: state.application.isNotificationOpen,
```

Comment:
Settings and Login screens use this I believe, wouldn't that get broken if removed?

Reply:
I doubt that, it is not being used in this reference

Reply:
giving it a second look just in case

Reply:
I am pretty confident it won't break anything, check the screenshot fro details
![Screenshot 2021-05-29 at 12 51 23 PM](https://user-images.githubusercontent.com/6298342/120062664-bfb94580-c07c-11eb-95da-bc1f33bf64eb.png)